src/components/AddPaymentMethodMenu.tsx

```diff
@@ -118,7 +116,10 @@ function AddPaymentMethodMenu({
                           {
                               text: translate('common.businessBankAccount'),
                               icon: Expensicons.Building,
-                              onSelected: () => onItemSelected(CONST.PAYMENT_METHODS.BUSINESS_BANK_ACCOUNT),
+                              onSelected: () => {
+                                  completePaymentOnboarding(CONST.PAYMENT_SELECTED.BBA);
```

Comment:
The workspace ID of the onboarding task can be -1 and user is navigated to the "Not here" page. https://github.com/Expensify/App/issues/52236 caught this.